### PR TITLE
Graph

### PR DESCRIPTION
**Pull request by ygao0719**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code

### CODE REVIEW

File: 401-code-challenges/Data-Structures/src/main/java/graph/Edge.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+public class Edge {
+    private static final int DEFAULT_WEIGHT = 1;
+    Node source;
+    Node destination;
+    int weight;
+
+    Edge(Node source, Node destination){
+        this(source,destination,DEFAULT_WEIGHT);
+    }
+    Edge(Node src, Node dest, int weig){
```
Comment (evanslaton) [inline 401-code-challenges/Data-Structures/src/main/java/graph/Edge.java:12]: Everywhere else you call it `source` and then here you call is `src`, remember that consistency will make it easier for those reading your code (including yourself in the future!)
